app/services/rapid_ready_for_decision/constants.rb

```diff
@@ -7,7 +7,7 @@ class Constants
         code: 7101,
         label: 'hypertension',
         flipper_name: 'hypertension',
-        sidekiq_job: 'RapidReadyForDecision::Form526HypertensionJob',
+        sidekiq_job: 'RapidReadyForDecision::Form526BaseJob',
```

Comment:
So is the idea here that if there is a disability the requires specific handling, it will get its own job class, but if it's generic _enough_ it will just use the base job?

Reply:
Pretty much. I would state it as: if the claim/disability requires special Sidekiq configurations that is different from `Form526BaseJob` (such as number of retries), then it can have it's own `sidekiq_job` class.

Reply:
Ok, I see the hypertension job getting deleted in Part 2